esx_service/utils/convert.py

```diff
@@ -24,7 +24,11 @@ def convert_to_MB(vol_size_str):
     """
 
     unit = vol_size_str[-2:].upper()
-    value = int(vol_size_str[:-2])
+    try:
```

Comment:
The current fix makes sense as it's consistent with the logic at line 41 - logging an error and returning 0. However, I checked the caller's logic in auth.py - we are not checking the invalid return value 0 at all. Ideally we should fix that as well to make it more robust.